### ISSUE TITLE
Log output to file?

### ISSUE THREAD

**Issue by erquhart** [tvooo/grunt-stylestats#3]
Any thoughts on adding an option to output stats to a file?

**Comment by Martinspire** [comment]
Yeah indeed, or even output like the website!

**Comment by tvooo** [comment]
Yes, I have a lot to do here. The grunt task also depends on a fairly old version of stylestats. I will see if I have time to work on this on the weekend, otherwise feel free to create pull requests.
